app/assets/stylesheets/application.bootstrap.scss

```diff
@@ -1,2 +1,13 @@
 @import 'bootstrap/scss/bootstrap';
 @import 'bootstrap-icons/font/bootstrap-icons';
+
+.card-container {
+    width: 400px; /* Adjust the width as needed */
+    height: 50%; /* Adjust the height as needed */
+  }
+  
+  .product-image {
+    width: 400px; /* Make sure the image takes the full width of the container */
+    height: 400px; /* Adjust the height as needed */
+    object-fit: cover; /* This scales the image to cover the entire container while maintaining aspect ratio */
```

Comment:
Let's remove these comments. Adding this much details regarding each line makes the code more clutter filled rather than making it easier to understand.